docs/install/README.md

```diff
@@ -5,7 +5,7 @@ The following is an overview of the different installation methods available.
 ## Prerequisites
 
 Kata Containers requires nested virtualization or bare metal. Check 
-[hardware requirements](/src/runtime/README.md#hardware-requirements) to see if your system is capable of running Kata 
+[hardware requirements](/README.md#hardware-requirements) to see if your system is capable of running Kata 
```

Comment:
I think if it's within the same file you don't need the file name
```suggestion
[hardware requirements](#hardware-requirements) to see if your system is capable of running Kata 
```

Reply:
Actually, the file `docs/install/README.md` does not have the `hardware-requirements` section for now.

Reply:
Ah sorry, I misunderstood, if you are pointing to the top-level README.md then you'll want `./../..//README.md#hardware-requirements`

Reply:
Updated.

Reply:
Thanks - looks good now!